src/Expensify.js

```diff
@@ -134,7 +135,8 @@ class Expensify extends PureComponent {
     }
 
     componentWillUnmount() {
-        AppState.removeEventListener('change', this.initializeClient);
+        if (!this.appStateChangeListener) { return; }
```

Comment:
Could you please expand this to multiple lines?
```suggestion
        if (!this.appStateChangeListener) {
        	return;
        }
```